docs/scripts/gen-faq.js

```diff
@@ -0,0 +1,174 @@
+// read.js
+const fs = require('fs')
+const yaml = require('js-yaml')
+const { Remarkable } = require('remarkable')
+const yamlPath = './docs/faq.yaml'
```

Comment:
Don't use static, relative file references. Replace with `path.resolve('path/to/file')`